From 862cbc625d15da9138a6a33242843e7abd4a5a8c Mon Sep 17 00:00:00 2001
From: Karthikeyan Kathirvel <quic_kathirve@quicinc.com>
Date: Thu, 27 Jan 2022 13:14:55 +0530
Subject: [PATCH] ath11k: adding temporary changes to match nl cmds

This change is to match the nl cmds and attr with the hostapd nl cmds
and attr respectively. This file should be removed once after the hostapd
pkg upgrade.

Signed-off-by: Karthikeyan Kathirvel <quic_kathirve@quicinc.com>
---
 include/uapi/linux/nl80211.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/include/uapi/linux/nl80211.h
+++ b/include/uapi/linux/nl80211.h
@@ -3183,12 +3183,11 @@ enum nl80211_attrs {
 
 	NL80211_ATTR_HE_MUEDCA_PARAMS,
 
-	NL80211_ATTR_MULTIPLE_BSSID_NON_TRANSMITTING,
 	NL80211_ATTR_MULTIPLE_BSSID_PARENT,
 	NL80211_ATTR_MULTIPLE_BSSID_INDEX,
 	NL80211_ATTR_MULTIPLE_BSSID_COUNT,
 	NL80211_ATTR_MULTIPLE_BSSID_IES,
-
+	NL80211_ATTR_MULTIPLE_BSSID_EMA,
 
 	NL80211_ATTR_OBSS_COLOR_BITMAP,
 
@@ -3209,6 +3208,8 @@ enum nl80211_attrs {
 
 	NL80211_ATTR_6G_REG_POWER_MODE,
 
+	NL80211_ATTR_MULTIPLE_BSSID_NON_TRANSMITTING,
+
 	/* add attributes here, update the policy in nl80211.c */
 
 	__NL80211_ATTR_AFTER_LAST,
